app/src/main/java/org/thunderdog/challegram/ui/SettingsController.java

```diff
@@ -862,6 +870,9 @@ public void onMyUserUpdated (final TdApi.User myUser) {
     }
     runOnUiThreadOptional(() -> {
       updateHeader();
+      if (setUserId(myUser)) {
+        adapter.updateValuedSettingById(R.id.btn_peer_id);
+      }
```

Comment:
Can't change during runtime (except being logged out, where it gets replaced to 0, in which case there's no need to update UI)